Reflection_Engine/Query/BHoMAssemblyList.cs

```diff
@@ -41,16 +41,6 @@ public static List<Assembly> BHoMAssemblyList()
             return m_BHoMAssemblies.Values.ToList();
         }
 
-        /***************************************************/
```

Comment:
I think this one is similar to my comment on the BHoM_UI PR regarding partial name matching. IT makes al lot of sense to have both `BHoMAssemblyList` and `AllAssemblyList` under the same file as they are basically dispatching to the `ExtractAllAssemblies` method below. Since the file was named `AssemblyList`, it actualy matches my recommendation of having the method names ending (or starting) with the file name instead of matching it exactly.